source/frontend/src/features/properties/map/project/update/UpdateProjectContainer.tsx

```diff
@@ -1,74 +1,93 @@
-        validationSchema={validationSchema}
-        onSubmit={handleSubmit}
-      >
-        {formikProps => (
-          <>
-            <Container>
-              <Section>
-                <SectionField label="Project name" required>
-                  <Input field="projectName" />
-                </SectionField>
-                <SectionField label="Project number">
-                  <Input field="projectNumber" placeholder="if known" />
-                </SectionField>
-                <SectionField label="Status">
-                  <Select
-                    field="projectStatusType"
-                    options={projectStatusOptions}
-                    placeholder="Select..."
-                  />
-                </SectionField>
-                <SectionField label="MoTI region" required>
-                  <Select field="region" options={projectRegionOptions} placeholder="Select..." />
-                </SectionField>
-                <SectionField label="Project summary">
-                  <MediumTextArea field="summary" />
-                </SectionField>
-              </Section>
-            </Container>
-          </>
-        )}
-      </Formik>
-    );
-  },
-);
+  const intialValues = ProjectForm.fromApi(project);
+  const projectStatusTypeCodes = getOptionsByType(API.PROJECT_STATUS_TYPES);
+  const regionTypeCodes = getByType(API.REGION_TYPES).map(c => mapLookupCode(c));
+
+  const handleSubmit = async (values: ProjectForm, formikHelpers: FormikHelpers<ProjectForm>) => {
+    try {
+      const updatedProject = values.toApi();
+      const response = await updateProject(updatedProject);
+
+      if (!!response?.id) {
+        formikHelpers?.resetForm();
+        if (typeof onSuccess === 'function') {
+          onSuccess();
+        }
+      }
+    } finally {
+      formikHelpers?.setSubmitting(false);
+    }
+  };
+
+  return (
+    <Formik<ProjectForm>
+      enableReinitialize
+      innerRef={formikRef}
+      initialValues={intialValues}
+      validationSchema={AddProjectYupSchema}
+      onSubmit={handleSubmit}
+    >
+      {formikProps => (
+        <>
+          <Container>
+            <Section>
+              <SectionField label="Project name" required={true}>
+                <Input field="projectName" />
+              </SectionField>
+              <SectionField label="Project number">
+                <Input field="projectNumber" placeholder="if known" />
+              </SectionField>
+              <SectionField label="Status">
+                <Select
+                  field="projectStatusType"
+                  options={projectStatusTypeCodes}
+                  placeholder="Select..."
+                />
+              </SectionField>
+              <SectionField label="MoTI region" required={true}>
+                <Select field="region" options={regionTypeCodes} placeholder="Select..." />
+              </SectionField>
+              <SectionField label="Project summary">
```

Comment:
Containers should not have UI elements when possible. Given these fields are mostly the same with the create section, reusing them would be preferable